src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/CompileExpressionsTests.cs

```diff
@@ -944,5 +946,172 @@ .maxstack  1
                         """);
                 });
         }
+
+        [WorkItem(66109, "https://github.com/dotnet/roslyn/issues/66109")]
+        [Fact]
+        public void SequencePointsMultipleDocuments()
+        {
+            var sourceA =
+@"partial class Program
+{
+    private int x = 1;
+    private void F()
+    {
+    }
+}";
+            var sourceB =
+@"partial class Program
+{
+    private int y = 2;
+    public Program()
+    {
+        F();
+        int z = x + y;
+#line 999
```

Comment:
What's the effect of this line directive on the test?

Reply:
I see that we create a context at line 999, just curious if there's something additional being tested by doing this instead of choosing an "original" line number that is within the method.

Reply:
The line directive was just used by the test to find an IL offset that included the local scope. I've changed the test to use an explicit offset instead, to avoid a chance that line directives affect the behavior.